### DISCUSSION TITLE
Which version of MSS configurator based on HSS version ?

### DISCUSSION THREAD

**Discussion by JV73** [polarfire-soc/polarfire-soc-documentation#320] [HSS and U-Boot] [answered]
Is there a document/table somewhere that can inform me as to which version of MSS_configurator must be used with a given version of the HSS application ?  I am currently using HSS v2022.09 (0.99.32-dev-build) and MSS_Configurator.exe from Libero v2022.1.  Are these compatible ?

**Comment by hughbreslin** [comment, marked as answer]
Hey @JV73 there shouldn't be compatibility issues between versions of XML produced. There are updates and enhancements to the values produced by the MSS configurator for things like DDR training. For example with the MSS configurator pre 2022.3 you need to modify values in the default [DDR profile](https://github.com/polarfire-soc/icicle-kit-reference-design#mss-lpddr4-configuration) which are now the default in the latest.

Best advice is use the latest version of the MSS configurator. The header files from the XML should always compile.
